test/webaudio/audioworklet_2x_in_hard_pan.c

```diff
@@ -65,6 +75,13 @@ void processorCreated(EMSCRIPTEN_WEBAUDIO_T context, bool success, void* data) {
   // Register a click to start playback
   emscripten_set_click_callback(EMSCRIPTEN_EVENT_TARGET_DOCUMENT, WA_2_VOIDP(context), false, &onClick);
 
-  // Register the counter that exits the test after one second of mixing
+#ifdef TEST_AND_EXIT
+  // Register the counter that exits the test after one second of playback
   emscripten_set_timeout_loop(&playedAndMixed, 16, NULL);
+#endif
+}
+
+// This implementation has no custom start-up requirements
+EmscriptenStartWebAudioWorkletCallback getStartCallback(void) {
```

Comment:
Why does this need to be a function that returns a function?  Why not just declare a function called `startCallback`?

Reply:
It was because most tests return and use this default function in the shared code:

https://github.com/emscripten-core/emscripten/blob/a55981e08e59d14774f81f550a83904fa197f099/test/webaudio/audioworklet_test_shared.inc#L78

Whereas one needed a custom implementation, so this was the cleanest way to keep most of the code shared.

Reply:
(Cleanest way and still keeping it C)